File: kits/java/lux/City.java
package lux;

import java.util.ArrayList;

public class City {
  public String cityid;
  public int team;
  public double fuel;
  public ArrayList<CityTile> citytiles = new ArrayList<>();
  private double lightUpKeep = 0.0;
  public City(int teamid, String cityid, double fuel, double lightUpKeep) {
    this.cityid = cityid;
    this.team = teamid;
    this.fuel = fuel;
    this.lightUpKeep = lightUpKeep;
  }
  public CityTile _add_city_tile(int x, int y, double cooldown) {
    CityTile ct = new CityTile(this.team, this.cityid, x, y, cooldown);
    this.citytiles.add(ct);
    return ct;
  }
  public double getLightUpkeep() {
    return this.lightUpKeep;
  }
}

File: kits/kotlin/simple/bot/src/main/kotlin/luxaibot/lux/City.kt
package luxaibot.lux

import java.util.ArrayList

class City(var team: Int, var cityid: String, var fuel: Double, lightUpKeep: Double) {
    var citytiles: ArrayList<CityTile> = ArrayList<CityTile>()
    var lightUpkeep = 0.0

    fun _add_city_tile(x: Int, y: Int, cooldown: Double): CityTile {
        val ct = CityTile(team, cityid, x, y, cooldown)
        citytiles.add(ct)
        return ct
    }

    init {
        lightUpkeep = lightUpKeep
    }
}
